src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/3.1.0.ts

```diff
@@ -217,6 +269,46 @@ export class USDTieredSTO_3_1_0 extends USDTieredSTOBase_3_1_0 {
     return typedResult;
   };
 
+  /**
+   * Return symbol of the denominated currency
+   * @return token symbol string
+   */
+  public denominatedCurrency = async () => {
+    const result = await (await this.contract).denominatedCurrency.callAsync();
+    return bytes32ToString(result);
+  };
+
+  /**
+   * Modifies oracle
+   */
+  public modifyOracles = async (params: ModifyOraclesParams): Promise<PolyResponse> => {
+    assert.assert(
+      await this.isCallerTheSecurityTokenOwner(params.txData),
+      ErrorCode.Unauthorized,
+      'The caller must be the ST owner',
+    );
+    assert.assert(params.denominatedCurrencySymbol !== '', ErrorCode.InvalidData, 'Invalid denominatedCurrencySymbol');
+    assert.assert(
+      params.customOracleAddresses.length === 2,
+      ErrorCode.InvalidLength,
+      'Invalid customOracleAddresses length',
+    );
+    return (await this.contract).modifyOracles.sendTransactionAsync(
+      params.customOracleAddresses,
+      stringToBytes32(params.denominatedCurrencySymbol),
+      params.txData,
+      params.safetyFactor,
+    );
+  };
```

Comment:
There is one more assert here in the contract. The denominated currency can change only if the STO hasn't started yet.

Also it can be reset to default Polymath oracles and `USD` as denominated currency. It must accept that case too, where `customOracleAddresses.length` is 0 and `denominatedCurrencySymbol` is `USD`.

Finally, if ETH is enabled as fundRaiseType, the first customOracleAddresses can't be `address(0)`. Same for the second If POLY is enabled. This allows issuer to send ZERO_ADDRESS as customOracleAddresses when they are not required.